src/foam/nanos/google/api/auth/GoogleApiAuthService.js

```diff
@@ -54,7 +55,9 @@ foam.CLASS({
         'java.io.IOException'
       ],
       javaCode: `
-        GoogleApiCredentials credentialsConfig = (GoogleApiCredentials) ((DAO)getX().get("googleApiCredentialsDAO")).find(((AppConfig)x.get("appConfig")).getUrl());
+        HttpServletRequest req = (HttpServletRequest)x.get(HttpServletRequest.class);
+        GoogleApiCredentials credentialsConfig = (GoogleApiCredentials) ((DAO)getX().get("googleApiCredentialsDAO")).find(req.getServerName());
```

Comment:
so serverName is just 'localhost' rather than http://localhost:8080, for example. This is what you want?

Reply:
yep